src/common-rest-api/src/main/java/org/niis/xroad/restapi/config/NotificationApiDisableSessionTimeoutFilter.java

```diff
@@ -55,14 +55,14 @@
 @Slf4j
 @Configuration
 @Order(Ordered.HIGHEST_PRECEDENCE)
-public class SessionTimeoutFilter extends GenericFilterBean {
+public class NotificationApiDisableSessionTimeoutFilter extends GenericFilterBean {
 
     private static final String ORIGINAL_MAX_INACTIVE_ATTR =
             "originalMaxInactiveInterval";
 
     @Override
     public void doFilter(ServletRequest request, ServletResponse response,
-            FilterChain chain) throws IOException, ServletException {
+                         FilterChain chain) throws IOException, ServletException {
```

Comment:
Does this not cause checkstyle error now? Indentation should be 8 instead of 21 according to the checkstyle rules

Reply:
fixed 
1887dc0